spec/spec_helper.rb

```diff
@@ -1,3 +1,9 @@
+require "capybara/rspec"
```

Comment:
we can move configurations related to capybara to `rails_helper.rb` because `spec_helper.rb` should contains configurations for unit tests only. Another words, `spec_helper.rb` should be simple